ideascube/tests/test_views.py

```diff
@@ -363,7 +363,9 @@ def test_staff_can_set_password(staffapp, client, user):
     response = form.submit(status=302)
     assert response.location.endswith(
         reverse('user_detail', kwargs={'pk': user.pk}))
-    assert user_model.objects.get(pk=user.pk).password != old_password
+    dbuser = user_model.objects.get(pk=user.pk)
+    assert not dbuser.check_password(old_password)
```

Comment:
`old_password` will probably be encrypted, so `old_password` will probably check with nothing. We should use the plain old `!=` here.